### PR TITLE
how to create new test data documentation

### PR DESCRIPTION
**Pull request by uzfarid**

### CODE REVIEW

File: jmeter-icap/instructions/How to prepare the test data.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Once test files are in s3 bucket then we have to set the load test framework to use new CSV file.
+
+The framework UI interface allows to upload new CSV test data metada file. It is in UI -> Setup page. 
+
+![setup-dataset](img/how-to-change-data-set-metadata.png)
+
+Select the new CSV and click on Submit Configurations button.
+
+**Note**: Upload Test Data to S3 bucket feature uploads only the test data provided by the framework internally and not any new test data.
```
Comment (NadHodeGW) [inline jmeter-icap/instructions/How to prepare the test data.md:38]: Im not too sure of the message here, can you clarify.

File: jmeter-icap/instructions/How to prepare the test data.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+The load test framework will send files for processing and compare rebuild hash with the one in the excel file. 
+
+Once the CSV file containing above meta-data is ready, then actual test files should be uploaded to the test data bucket. It is better to prepare them locally and then upload to the s3 bucket. The structure of the test data folders should be:
+
+bucket->fileType->subFolder->actualFile
+
+Once test files are in s3 bucket then we have to set the load test framework to use new CSV file.
+
+The framework UI interface allows to upload new CSV test data metada file. It is in UI -> Setup page.
```
Comment (NadHodeGW) [inline jmeter-icap/instructions/How to prepare the test data.md:32]: typo in 'metada' should be 'metadata'

File: jmeter-icap/instructions/How to prepare the test data.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+**original_hash**: the file hash calculated before processing with SHA-256 algorithm
+
+**rebuild_hash**: the out file hash calculated with SHA-256 algorithm
+
+(https://github.com/k8-proxy/aws-jmeter-test-engine/blob/master/jmeter-icap/scripts/ICAP-Direct-File-Processing/gov_uk_files.csv is example comprehensive meta data CSV file, but minimum required fields are mentioned above)
+
+The load test framework will send files for processing and compare rebuild hash with the one in the excel file. 
+
+Once the CSV file containing above meta-data is ready, then actual test files should be uploaded to the test data bucket. It is better to prepare them locally and then upload to the s3 bucket. The structure of the test data folders should be:
```
Comment (NadHodeGW) [inline jmeter-icap/instructions/How to prepare the test data.md:26]: typo in 'metada' should be 'metadata'

File: jmeter-icap/instructions/How to prepare the test data.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# How to prepare test data ?
+
+This instruction assumes basic Load Test framework setup is done using the main instructions in https://github.com/k8-proxy/aws-jmeter-test-engine/blob/master/README.md
+
+The framework comes to setup sample test data and the data should be uploaded to the s3 test data bucket as described in documentation in above link.
```
Comment (NadHodeGW) [inline jmeter-icap/instructions/How to prepare the test data.md:5]: is this meant to say: ' The framework comes with sample test data'?

**Review by NadHodeGW — CHANGES_REQUESTED** [review]

File: jmeter-icap/instructions/How to prepare the test data.md (hunk 1 above)
Comment (uzfarid) [reply jmeter-icap/instructions/How to prepare the test data.md:38]: it is as it says. what other clarification is needed.

Comment (jpoolGW) [reply jmeter-icap/instructions/How to prepare the test data.md]: Can be rephrased as
"Upload Test Data option refers only to specific sample test data contained within this repo"

**Review by jpoolGW — APPROVED** [review]

**Pull request merged**